content/sources/workflow.adoc

```diff
@@ -40,18 +42,39 @@ Branching model are 6 kinds of branch :
 
 === Develop Branch
 
-Develop branch is the production ready branch which is, in general case, the CI/CD production branch.
+For meeds-io repositories, develop branch is the branch containing all finished and tested features.
+For exoplatform repositories, develop branch is the production ready branch which is, in general case, the CI/CD production branch.
 
 It contains the latest delivered development changes. This is our backbone where all the different fix and new feature are mixed with each other.
 
+For meeds-io repositories, when a member finish a feature, he merges it on develop branch from his develop-xxx branch. Then, on a regularly base, a rebase is done to propagate theses new developments on other members develop branch :
+
 [[git-workflow-develop-branch]]
-.Git Workflow - Develop Branch
-image::git-workflow-develop-branch.png[Git Workflow - Develop Branch, 600, align=center, link={imagesdir}/git-workflow-develop-branch.png]
+.Git Workflow - Rebase Develop Branch
+image::git-worflow-rebase-develop.png[Git Workflow - Rebase Develop Branch, 800, align=center, link={imagesdir}/git-worflow-rebase-develop.png]
```

Comment:
We will have to review this workflow since another proposition was made and applied by @hbenali